### PR TITLE
[171] Fix for to_unixtime() issue

### LINKED ISSUE #171: Bug in to_unixtime()
### Software Environment:
- OpenLooKeng version (source or binary):

- OS platform & distribution (eg., Linux Ubuntu 16.04):

- Java version:
### Describe the current behavior

1.  `to_unixtime()` without `time zone`, by default use `UTC`.
image

2.  Different result on UI.
image

3.  Same result with different timezones.
image

### Describe the expected behavior

1.  There should be a configuration to update `time zone`, which would be used by `to_unixtime()`.
2.  UI and CLI output should match.
3.  Result should be according to time zone

### Steps to reproduce the issue
1.
2.
3.

### Special notes for this issue

### PR DESCRIPTION
**Pull request by aman-omer**
### What type of PR is this?

/kind bug

### What does this PR do / why do we need it:

Fixes bug in `to_unixtime` function.

### Which issue(s) this PR fixes:

Fixes #171 

### Special notes for your reviewers:

Added session property `time_zone_id`.

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/operator/scalar/DateTimeFunctions.java [hunk 1]
```diff
@@ -175,16 +176,23 @@ public static long fromUnixTime(@SqlType(StandardTypes.DOUBLE) double unixTime,
 
     @ScalarFunction("to_unixtime")
     @SqlType(StandardTypes.DOUBLE)
-    public static double toUnixTime(@SqlType(StandardTypes.TIMESTAMP) long timestamp)
+    public static double toUnixTime(ConnectorSession session, @SqlType(StandardTypes.TIMESTAMP) long timestamp)
```
Comment (Nitin-Kashyap) [inline presto-main/src/main/java/io/prestosql/operator/scalar/DateTimeFunctions.java:179]: Needs test cases

Comment (aman-omer) [reply presto-main/src/main/java/io/prestosql/operator/scalar/DateTimeFunctions.java:179]: Added

**Comment by Nitin-Kashyap** [comment]
lgtm

**Pull request merged**